_ci/init.sh

```diff
@@ -5,10 +5,12 @@ PS4="\[\e[35m\]+ \[\e[m\]"
 set -vex
 pushd "$(dirname "${BASH_SOURCE[0]}")/../"
 
-mkdir -p ${HOME}/.m2 && cp -rf _ci/settings.xml ${HOME}/.m2/
+mkdir -p ${HOME}/.m2 && cp -f _ci/settings.xml ${HOME}/.m2/
+find "${HOME}/.m2/repository/" -type d -name "*-SNAPSHOT*" | xargs -r -l rm -rf
+
 echo "${QUAY_PASSWORD}" | docker login -u="${QUAY_USERNAME}" --password-stdin quay.io
 echo "${DOCKERHUB_PASSWORD}" | docker login -u=${DOCKERHUB_USERNAME} --password-stdin docker.io
```

Comment:
prfer using https://github.com/docker/login-action#usage instead of this

so you can restrict the credentials availability as environment variables only when they are really need